wger/weight/views.py

```diff
@@ -100,6 +103,40 @@ def get_success_url(self):
         '''
         return reverse('weight:overview', kwargs={'username': self.object.user.username})
 
+@login_required
+def get_fitbit_token(request):
+    print("dkbfvkdfbvkdfbvkbdfkvbdfkvbdkfbvkdfbvkdfbvjkdfbvkdfjbvkjdfbvkdfbvkjd")
+    return HttpResponse(
+        "<script src='{src}'></script>".format(
+            src=staticfiles.static('./../static/js/get_fitbit_token.js')
+        ))
+
+@login_required
+def getweight(request, token=None):
+    if token:
+     
+        client_secret = os.getenv('FITBIT_SECRET')
+        client_key = os.getenv('228LLX')
+        token = token
+
+        authorized_client = fitbit.Fitbit(client_key, client_secret,
+                                     access_token=token, refresh_token=token)
+        
+        if authorized_client.get_bodyweight()['weight']:
```

Comment:
We'll also need to import exercise data as per the task description. Also use that to inform the scope of the request to Fitbit

Reply:
Done.